### ISSUE TITLE
Inconsistent dashboard.ps1 merge issues when multiple modules define dashboards

### ISSUE THREAD

**Issue by jpbruckler** [ironmansoftware/powershell-universal#2511]
### Version

4.0.2

### Severity

High

### Steps to Reproduce

This feels like a "high", my test environments are either new installs from 4.0.2, or upgrades to 4.0.3 from 4.0.2 so may not be widespread. I'm working on an Identity Management module which provides a dashboard using a module-scoped .universal file with a dashboards.ps1 file. I noticed that when adding other modules that also provide apps via dashboards.ps1 that not all the apps are loaded/activated.

This can be replicated by installing Universal.App.ActiveDirectory and Universal.App.PowerGUI. The only app available will be the ActiveDirectory app, PowerGUI app till not show in the app list, and navigating to /powergui will result in a 404.

I have noted that in 4.0.3 on initial install that both apps are available, but that on restart of the PowerShell Universal service only the ActiveDirectory app will show up. It seems to only load the first module based on alphabetical sorting.

Manually adding the calls to `New-PSUDashboard` to the server level dashboards.ps1 will load the app.

### Expected behavior

```console
All modules that add apps will be loaded and their app is available for use.
```

### Actual behavior

```console
Only the first module that provides an app is loaded.
```

**Comment by jpbruckler** [comment]
After some further testing, while manually adding the calls to New-PSUDashboard to the server level dashboards.ps1 will load the app, if the module provides variables via variables.ps1 those variables disappear from the console. So it _seems like_ merging the .universal directives isn't working at least how I the user understand them to work.

**Comment by adamdriscoll** [comment]
I can reproduce this. The module load was throwing an exception during the first app load that would cause it to fail to load any additional modules are that point. We'll have a fix in for 4.0.3.
